golang/cosmos/app/upgrade.go

```diff
@@ -11,11 +11,8 @@ import (
 )
 
 var upgradeNamesOfThisVersion = []string{
-	"UNRELEASED_BASIC", // no-frills
-	"UNRELEASED_A3P_INTEGRATION",
-	"UNRELEASED_main",
-	"UNRELEASED_devnet",
-	"UNRELEASED_REAPPLY",
+	"agoric-upgrade-17",
+	"agorictest-upgrade-17",
```

Comment:
Ah I now remember another change we decided in the last upgrade: drop the `agorictest` upgrade handler names, and instead make sure we have a `agoric-upgrade-17-2` available to re-apply the upgrade if necessary.

I suppose this may require a change in the release notes script as well.

https://github.com/Agoric/agoric-sdk/blob/agoric-upgrade-16/golang/cosmos/app/upgrade.go#L14-L15